zeroconf.py

```diff
@@ -1416,7 +1416,8 @@ def __init__(self, type_: str, name: str, address: bytes = None, port: int = Non
         else:
             self.server = name
         self._properties = {}  # type: ServicePropertiesType
-        self._set_properties(properties)
+        # Verify that properties is not None. Passing None breaks the object's state. 
+        self._set_properties(properties if properties else b'')
```

Comment:
Those two lines are not necessary I think.

Reply:
As the comment says (it is fine to remove it), if the properties are None, the unit tests break. It is weird, since they were working before this modification. Any idea why it breaks? Should the unit tests be changed? What implications it has on legacy code?

Reply:
Can you show the breakage?

Reply:
Now it didn't broke... I'm sorry for the nuisance! Amended the last commit. Please check if it is ok.